src/scales/scale.time.js

```diff
@@ -36,6 +36,19 @@ module.exports = function(Chart) {
 		},
 		ticks: {
 			autoSkip: false
+		},
+		majorTicks: {
```

Comment:
I liked to have all tick options scoped under the `ticks` key, but having it duplicated under `majorTicks` doesn't look ideal but also not obvious that it overrides the `ticks` options. Not sure either about this alternative, but what do you think about:

```javacript
ticks: {
    // common tick options
    // ...
    minor: {
        // overrides for minor ticks only
    },
    major: {
        // overrides for major ticks only
    }
}
```

> `ticks.minor: false`: don't show minor ticks
> `ticks.major: false`: don't show major ticks